### DISCUSSION TITLE
Not working Arbitrary background Images with specific breakpoints.

### DISCUSSION THREAD

**Discussion by zamarproject** [tailwindlabs/tailwindcss#10656] [Ideas]
I´ve been striving to make it work this code, but is not working to be responsive with specific breakpoints as it should :

bg-[url('images/xs/{{ $background->url }}')]

md:bg-[url('images/md/{{ $background->url }}')]
xl:bg-[url('images/xl/{{ $background->url }}')]

Only is working without a specific breakpoint  

It would be great to have this feature

**Comment by adamwathan** [comment]
Hey! The issue here is that you are using dynamic class names:

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

To achieve what you're trying to do I'd use a combination of inline styles and Tailwind utilities that reference CSS variables.

Assuming this is something like Blade based on the syntax you've shared, it would be something like this:

```html
<div
  class="bg-[url:var(--bg-image)] md:bg-[url:var(--bg-image-md)] xl:bg-[url:var(--bg-image-xl)]" 
  style="--bg-image: url('images/xs/{{ $background->url }}'); --bg-image-md: url('images/md/{{ $background->url }}'); --bg-image-xl: url('images/xl/{{ $background->url }}')"
>
```

**Reply by zamarproject** [reply]
Thank you so much! I truly appreciate it a lot.
Before to your suggestion, I started using <style> and using media-queries before to the element, but having errors in the console. 
This works great!
Thank you so much ! Greetings!
